recipes/qt/5.x.x/conanfile.py

```diff
@@ -0,0 +1,710 @@
+        "with_doubleconversion": [True, False],
+        "with_freetype": [True, False],
+        "with_fontconfig": [True, False],
+        "with_icu": [True, False],
+        "with_harfbuzz": [True, False],
+        "with_libjpeg": ["libjpeg", "libjpeg-turbo", False],
+        "with_libpng": [True, False],
+        "with_sqlite3": [True, False],
+        "with_mysql": [True, False],
+        "with_pq": [True, False],
+        "with_odbc": [True, False],
+        "with_libalsa": [True, False],
+        "with_openal": [True, False],
+        "with_zstd": [True, False],
+
+        "gui": [True, False],
+        "widgets": [True, False],
+
+        "device": "ANY",
+        "cross_compile": "ANY",
+        "sysroot": "ANY",
+        "config": "ANY",
+        "multiconfiguration": [True, False],
+    }
+    options.update({module: [True, False] for module in _submodules})
+
+    no_copy_source = True
+    default_options = {
+        "shared": False,
+        "commercial": False,
+        "opengl": "desktop",
+        "with_vulkan": False,
+        "openssl": True,
+        "with_pcre2": True,
+        "with_glib": False,
+        # "with_libiconv": True, # QTBUG-84708
+        "with_doubleconversion": True,
+        "with_freetype": True,
+        "with_fontconfig": True,
+        "with_icu": True,
+        "with_harfbuzz": False,
+        "with_libjpeg": "libjpeg",
+        "with_libpng": True,
+        "with_sqlite3": True,
+        "with_mysql": True,
+        "with_pq": True,
+        "with_odbc": True,
+        "with_libalsa": False,
+        "with_openal": True,
+        "with_zstd": True,
+
+        "gui": True,
+        "widgets": True,
+
+        "device": None,
+        "cross_compile": None,
+        "sysroot": None,
+        "config": None,
+        "multiconfiguration": False,
+    }
+    default_options.update({module: False for module in _submodules})
+
+    requires = "zlib/1.2.11"
+    short_paths = True
+
+    def export(self):
+        self.copy("qtmodules%s.conf" % self.version)
+
+    def build_requirements(self):
+        if tools.os_info.is_windows and self.settings.compiler == "Visual Studio":
+            self.build_requires("jom/1.1.3")
+        if self.options.qtwebengine:
+            self.build_requires("ninja/1.10.2")
+            # gperf, bison, flex, python >= 2.7.5 & < 3
+            if self.settings.os != "Windows":
+                self.build_requires("bison/3.7.1")
+                self.build_requires("gperf/3.1")
+                self.build_requires("flex/2.6.4")
+
+            # Check if a valid python2 is available in PATH or it will failflex
+            # Start by checking if python2 can be found
+            python_exe = tools.which("python2")
+            if not python_exe:
+                # Fall back on regular python
+                python_exe = tools.which("python")
+
+            if not python_exe:
+                msg = ("Python2 must be available in PATH "
```

Comment:
does it still require python2? it's EOL already

Reply:
yes https://chromium.googlesource.com/chromium/src/+/master/docs/linux/build_instructions.md#system-requirements
https://wiki.qt.io/QtWebEngine/How_to_Try#Building